virttest/qemu_monitor.py

```diff
@@ -2419,7 +2419,7 @@ def sendkey(self, keystr, hold_time=1):
         """
         return self.human_monitor_cmd("sendkey %s %s" % (keystr, hold_time))
 
-    def migrate(self, uri, full_copy=False, incremental_copy=False, wait=False):
+    def migrate(self, uri, full_copy=None, incremental_copy=None, wait=False):
```

Comment:
Does this new usage also work on the hmp?

Reply:
> The docstring may be updated with the new parameter definition.

Good suggestion! I added the following to note the users, thanks!
```
(removed since qemu 9.1, don't use)
```